spec/support/wait_for_ajax.rb

```diff
@@ -0,0 +1,15 @@
+module WaitForAjax
+  def wait_for_ajax
```

Comment:
We gotta avoid using this. `wait_for_ajax` relies on the fact that we are using `jQuery` which isn't the case for the v2 board.

Reply:
Ok, I'll remove this and try the `wait_spinner` and see if it works.